Add VGMdb import script

augustjanse opened the pull request:
Adds an import script using the VGMdb.info API. Works for the most part, but could of course be further refined. Fixes #81.

murdos commented on vgmdb_importer.user.js:
```diff
@@ -0,0 +1,263 @@
+ * Returns a MusicBrainz format based on a VGMdb media format.
+ */
+function mapFormat(mediaFormat) {
+    return mediaFormat;
```
It seems that at least `Digital` should be converted to `Digital Media`

augustjanse replied:
Whoops, meant to fill that in later on. Thanks!

murdos commented:
Thanks! It's ready to be merged for you, or should I wait for further refinements?

augustjanse commented:
I fixed the `mapFormat()` function and added barcodes as well (after asking the API owner to add them to the API). So it's ready to be merged, probably won't do any more work unless someone asks for it.
